backstory/pom.xml

```diff
@@ -35,7 +35,34 @@
       <artifactId>appengine-api-1.0-sdk</artifactId>
       <version>1.9.59</version>
     </dependency>
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
```

Comment:
I think mockito-core is more recent. mockito-all 1.9.5 is from 2012.

Reply:
I updated it to mockito-core 3.4.4, and I am fixing it.

Reply:
Done